Frontend/CarRentalAdmin/src/app/auth-interceptor.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpRequest, HttpHandler, HttpEvent, HttpInterceptor } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class AuthInterceptor implements HttpInterceptor {
+
+  intercept(req: HttpRequest<any>,
+            next: HttpHandler): Observable<HttpEvent<any>> {
+
```

Comment:
```suggestion
  intercept(
    request: HttpRequest<any>,
    next: HttpHandler
  ): Observable<HttpEvent<any>> {

```

for a better readability if there are a lot of parameters each of them should be placed in new line
please use full names for variables, it also improves readability
I recommend to avoid placing of code in the middle of line
you decided to make indents with 2 spaces, so if you have nesting please use 2 spaces